test/integration/examples/apiserver_test.go

```diff
@@ -334,9 +337,7 @@ func TestAggregatedAPIServer(t *testing.T) {
 	// this is ugly, but sleep just a little bit so that the watch is probably observed.  Since nothing will actually be added to discovery
 	// (the service is missing), we don't have an external signal.
 	time.Sleep(100 * time.Millisecond)
-	if _, err := aggregatorDiscoveryClient.Discovery().ServerResources(); err != nil {
-		t.Fatal(err)
-	}
+	assertWardleUnavailableDiscoveryError(t, err)
```

Comment:
The wardle API server was never available, because `Service` object is missing.
Kube-aggregator previously silently skipped unavailable API services in the `/apis` endpoint. With the new change, there is an error with details that are asserted in the `assertWardleUnavailableDiscoveryError` function.